app/services/data/update-claim-overpayment-status.js

```diff
@@ -32,6 +32,10 @@ module.exports = function (claim, overpaymentResponse) {
     updateClaim.RemainingOverpaymentAmount = overpaymentResponse.remaining
   }
 
+  if (updateClaim.OverpaymentReason > 250) {
+    updateClaim.OverpaymentReason = updateClaim.OverpaymentReason.substring(0, 250)
+  }
+
```

Comment:
I'm not sure that just truncating the field is the best idea - this gives the illusion that you can add more text than you really can (unless the text box is limited to only 250 characters)

Reply:
It looks like the text box is limited